pyproject.toml

```diff
@@ -42,7 +42,6 @@ maintainers = [
 classifiers = [
     "Development Status :: 4 - Beta",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
```

Comment:
Shouldn't 3.9 be also removed by default (cf https://scientific-python.org/specs/spec-0000/)

Reply:
I agree but I think we should do that cleanup in a different PR

Reply:
Fine by me :)